index.html

```diff
@@ -272,29 +272,42 @@ <h5 class="navbar__submenu-title--right">Students</h5>
         <section>
             <div class="welcome">
                 <div class="welcome__box">
-                    <h1 class="welcome__title">GEEKS UI, Premium LMS / Education HTML Template</h1>
+                    <h1 class="welcome__title"><span class="welcome__geeks-span">Geeks UI</span>, Premium LMS / Education HTML Template</h1>
```

Comment:
Do not use tag name as a BEM Element name - let's focus of why it is bolded text fragment: `welcome__name` or `welcome__brand`? Maybe you have better idea?

Reply:
Ok. `welcome__brand` fits pretty well here, I'll apply this